src/components/Offramp/Confirm.view.tsx

```diff
@@ -722,36 +733,46 @@ export const OfframpConfirmView = ({
-                                        (account) =>
-                                            account.account_identifier.replaceAll(/\s/g, '').toLowerCase() ===
-                                            offrampForm.recipient.replaceAll(/\s/g, '').toLowerCase()
-                                    )?.account_type === 'iban'
-                                        ? '$1'
-                                        : '$0.50'}
+                                    {appliedPromoCode
+                                        ? '$0'
+                                        : user?.accounts.find(
+                                                (account) =>
+                                                    account.account_identifier.replaceAll(/\s/g, '').toLowerCase() ===
+                                                    offrampForm.recipient.replaceAll(/\s/g, '').toLowerCase()
+                                            )?.account_type === 'iban'
+                                          ? '$1'
+                                          : '$0.50'}
                                     <span className="inline-flex items-center">
                                         <MoreInfo
-                                            text={`For ${
-                                                user?.accounts.find(
-                                                    (account) =>
-                                                        account.account_identifier
-                                                            .replaceAll(/\s/g, '')
-                                                            .toLowerCase() ===
-                                                        offrampForm.recipient.replaceAll(/\s/g, '').toLowerCase()
-                                                )?.account_type === 'iban'
-                                                    ? 'SEPA'
-                                                    : 'ACH'
-                                            } transactions a fee of ${
-                                                user?.accounts.find(
-                                                    (account) =>
-                                                        account.account_identifier
-                                                            .replaceAll(/\s/g, '')
-                                                            .toLowerCase() ===
-                                                        offrampForm.recipient.replaceAll(/\s/g, '').toLowerCase()
-                                                )?.account_type === 'iban'
-                                                    ? '$1'
-                                                    : '$0.50'
-                                            } is charged.`}
+                                            text={
+                                                appliedPromoCode
+                                                    ? 'Fees waived with promo code!'
```

Comment:
thought: this works fine if promo codes fully subsidize the cashout, which might not necessarily hold in the future 
- but it's a good approach for current situation :+1: